app/views/taxons/index.html.erb

```diff
@@ -19,8 +18,18 @@
       <tr>
         <td><%= taxon['title'] %></td>
         <td><%= taxon['content_id'] %></td>
-        <td><%= link_to 'View tagged content', taxon_path(taxon['content_id']), class: 'view-tagged-content' %></td>
-        <td><%= link_to 'Edit taxon', edit_taxon_path(taxon['content_id']) %></td>
+        <td>
+          <%= link_to 'View', taxon_path(taxon['content_id']),
+            class: 'view-tagged-content btn btn-primary' %>
+
+          <%= link_to 'Edit', edit_taxon_path(taxon['content_id']),
```

Comment:
"View" and "Edit" are links to pages, so they need to look like links ("Delete" is an action, so it's good that it's a button).